doc/crypt.tex

```diff
@@ -5549,6 +5549,22 @@ \subsection{Signature Generation}
 This function creates the same ECDSA signature as \textit{ecc\_sign\_hash} only the output format is different.
 The format follows \url{https://tools.ietf.org/html/rfc7518#section-3.4}, sometimes it is also called plain signature.
 
+\index{ecc\_sign\_hash_ex()}
+\begin{verbatim}
+int ecc_sign_hash_ex(const unsigned char *in,
+                           unsigned long  inlen,
+                           unsigned char *out,
+                           unsigned long *outlen,
+                              prng_state *prng,
+                                     int  wprng,
+                                     int  sigformat,
```

Comment:
this should be `ecc_signature_type sigformat,` (applies to the other parts of the docs as well

Reply:
Ah! Oops, I'll change that now.